tests/vmi_multus_test.go

```diff
@@ -1008,16 +1042,72 @@ var _ = Describe("[Serial]Macvtap", func() {
 
 		BeforeEach(func() {
 			macAddress := "02:03:04:05:06:07"
-			macvtapVMI = createCirrosVMIWithMacvtapDefinedMAC(virtClient, macvtapNetworkName, macAddress)
+			clientVMI = createCirrosVMIStaticIPRandomNode(virtClient, macvtapNetworkName, macAddress)
 		})
 
 		It("should be successful when the VMI MAC address is defined in its spec", func() {
 			By("starting the migration")
-			migration := tests.NewRandomMigration(macvtapVMI.GetName(), macvtapVMI.GetNamespace())
+			migration := tests.NewRandomMigration(clientVMI.GetName(), clientVMI.GetNamespace())
 			migrationUID := tests.RunMigrationAndExpectCompletion(virtClient, migration, migrationWaitTime)
 
 			// check VMI, confirm migration state
-			tests.ConfirmVMIPostMigration(virtClient, macvtapVMI, migrationUID)
+			tests.ConfirmVMIPostMigration(virtClient, clientVMI, migrationUID)
+		})
+
+		Context("with live traffic", func() {
+			var serverVMI *v1.VirtualMachineInstance
+			var serverIP string
+
+			getVMMacvtapIfaceIP := func(vmName string, namespace string, macAddress string) (string, error) {
+				var vmiIP string
+				err := wait.PollImmediate(time.Second, 2*time.Minute, func() (done bool, err error) {
```

Comment:
When this returns an error, we should fail on it. In the current state, would always fail on 1085 and never get to the line 1087 in case this wait fails.

Reply:
BTW I like this approach, looks neat.

Reply:
my bad, forgot to return the error.